src/main/java/frc/robot/commands/ReleaseCoral.java

```diff
@@ -3,8 +3,11 @@
 import edu.wpi.first.wpilibj2.command.Command;
 import frc.robot.subsystems.CoralGripper;
 
+import java.time.Instant;
+
 public class ReleaseCoral extends Command {
     public CoralGripper coralGripper;
+    Instant previousTime = null;
```

Comment:
don't use `Instant` as it uses the less trusty system clock. Use `Timer.getFPGATimestamp()` instead.